packages/financial-templates-lib/test/clients/ExpiringMultiPartyClient.js

```diff
@@ -3,17 +3,20 @@ const { parseFixed } = require("@ethersproject/bignumber");
 const winston = require("winston");
 
 const { interfaceName, MAX_UINT_VAL, ZERO_ADDRESS } = require("@uma/common");
+const { getTruffleContract } = require("@uma/core");
+
+const ABI_VERSION = "1.2.0";
```

Comment:
is `ABI_VERSION` a correctly descriptive variable name? It's not _just_ the ABIs that are changing between versions; it's the whole contract + the bytecode & implementation details. Would `CONTRACT_VERSION` not be more descriptive?

Reply:
hm yeah I think that'd be more accurate I'll make the change